chatspaceでメッセージの自動更新

aki800 opened the pull request:
#WHAT
chatspaceでメッセージの自動更新機能を非同期通信で実装した。

#WHY
別のユーザーが投稿したメッセージを自動で見れるようにするため。

メッセージ自動で更新される様子↓
https://i.gyazo.com/5ad5e11419d629c1c1a1bf0a40974b07.mp4

Commit:
Auto update messages

miyagit reviewed (commented):
インデントがところどころおかしいですね。
それとすべてではなくて差分だけをとってきて加えるような自動更新にしましょう。

Commit:
インデントと自動投稿機能の修正

#WHAT
インデントを修正した。
新たに加えられた投稿を自動的に取得するようにした。

#WHY
読み手に分かりやすくするため。
効率的に機能させるため。

aki800 commented:
インデントと自動更新を修正しました。

TOMARUMARU reviewed (commented):
htmlを生成しているfunctionはメッセージの非同期通信と同じものを使いましょう

Commit:
Htmlの生成を共通化

#WHAT
メッセージ自動更新の非同期通信で用いているhtmlと、メッセージ投稿の非同期通信で用いているhtmlを共通化。

#WHY
重複したコードを省いて見やすくするため。

aki800 commented:
htmlをメッセージの非同期通信で使ってたものと同じものにしました。

TOMARUMARU reviewed (approved):
LGTM
